README.md

```diff
@@ -48,6 +52,14 @@ List of supported problem daemons:
 | [KernelMonitor](https://github.com/kubernetes/node-problem-detector/tree/master/pkg/kernelmonitor) | KernelDeadlock | A problem daemon monitors kernel log and reports problem according to predefined rules. |
 
 # Usage
+## Override Apiserver Client Configuration
+* `-apiserver-override`  
+`apiserver-override` command line argument can customize how to generate a Kubernetes ApiServer
```

Comment:
Rephrase the description a bit.

An URI parameter used to customize how node-problem-detector connects the apiserver. The format is the same with the `source` flag of Heapster:
```
http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&userServiceAccount=false&auth=&insecure=
```